bin/deploy.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+set -e
+
+export INGRESS_INTERNAL_ANNOTATIONS=$HOF_CONFIG/ingress-internal-annotations.yaml
+export INGRESS_EXTERNAL_ANNOTATIONS=$HOF_CONFIG/ingress-external-annotations.yaml
+export CONFIGMAP_VALUES=$HOF_CONFIG/configmap-values.yaml
+export NGINX_SETTINGS=$HOF_CONFIG/nginx-settings.yaml
+
+kd='kd --insecure-skip-tls-verify --timeout 10m --check-interval 10s'
+
+if [[ $1 == 'tear_down' ]]; then
+  export KUBE_NAMESPACE=$BRANCH_ENV
+  export DRONE_SOURCE_BRANCH=$(cat /root/.dockersock/branch_name.txt)
+
+  $kd --delete -f kube/configmaps/configmap.yml
+  $kd --delete -f kube/redis -f kube/app
+  echo "Torn Down UAT Branch - eec-$DRONE_SOURCE_BRANCH.internal.branch.sas-notprod.homeoffice.gov.uk"
+  exit 0
+fi
+
+export KUBE_NAMESPACE=$1
+export DRONE_SOURCE_BRANCH=$(echo $DRONE_SOURCE_BRANCH | tr '[:upper:]' '[:lower:]' | tr '/' '-')
+
+if [[ ${KUBE_NAMESPACE} == ${BRANCH_ENV} ]]; then
+  $kd -f kube/configmaps -f kube/certs
+  $kd -f kube/redis -f kube/app
+elif [[ ${KUBE_NAMESPACE} == ${UAT_ENV} ]]; then
+  $kd -f kube/configmaps/configmap.yml -f kube/app/service.yml
+  $kd -f kube/app/networkpolicy-internal.yml -f kube/app/ingress-internal.yml
+  $kd -f kube/app/networkpolicy-external.yml -f kube/app/ingress-external.yml
+  $kd -f kube/redis -f kube/app/deployment.yml
+elif [[ ${KUBE_NAMESPACE} == ${STG_ENV} ]]; then
+  $kd -f kube/configmaps/configmap.yml
+  $kd -f kube/redis
+  $kd -f kube/app
+
```

Comment:
Seems a little light in terms of deployments in STG...is this correct?

Reply:
Yes that's right. We are deploying all the objects in those folders for Staging.